script/Ciel/Day 14/14_01B_03_CIEL14B_6.txt

```diff
@@ -79,7 +80,7 @@ It seems like the Luminary was only a couple of steps from reaching the school.
 -- Page 289, Offset 41098.
 -- 「―――なんだ。最後の最後に、悪運に恵まれたのか」
 "―――Would you look at that.
- Looks like I was blessed with the devil's own luck right at the very end."
+ Seems like I was blessed with the devil's own luck right at the very end."
```

Comment:
I find this line weirdly long.
Maybe the first one can be cut down a bit, at least.
>Well, well.
>Seems like...